### ISSUE TITLE
[BUG] `waitForActive` to `true` makes Dynamoose hang

### ISSUE THREAD

**Issue by Drarig29** [dynamoose/dynamoose#1402]
### Summary:

Although the type of `waitForActive` says it can be a boolean, setting it to `true` makes Dynamoose hang.

Workaround: `waitForActive: { enabled: true }` works.

### Environment:

Operating System:
Operating System Version:
Node.js version (`node -v`): v14.17.6
NPM version: (`npm -v`): 8.5.5
Dynamoose version: 3.0.0-beta.1

**Comment by fishcharlie** [comment]
I tried to reproduce this using the following code, and didn't receive any hangs when running 3.0.0-beta.1.

```js
(async () => {
	const dynamoose = require("dynamoose");

	dynamoose.aws.ddb.local();

	const schema = new dynamoose.Schema({
		"id": String
	});

	const Model = dynamoose.model("Test", schema);

	new dynamoose.Table("Test", [Model], {
		"waitForActive": true
	});

	console.log("Done");
})();
```

**Comment by Drarig29** [comment]
Ok, maybe I didn't scope the right problem but I assure you that it started to work when I put waitForActive to `{ enabled: true }`. Maybe it was only for an actual remote dynamodb connection (not a local one). 

I was also using `create: true` and `update: true`

**Comment by fishcharlie** [comment]
Thank you for your issue. This project is successful due to the reports and help we get from the community in improving the project.

This project has a lot of guidelines that help ensure the project continues to run smoothly and project maintainers aren't burdened with a lot of back and forth discussion in order to advance the project and resolve issues.

**Sadly it has been determined that your issue does not follow those guidelines.**

This could be due to the following *checked* items (unchecked items were satisfied, found irrelevant, or not checked):

General

- [ ] Issue template not filled out
- [ ] Issue is unclear
- [ ] Issue doesn't explain what has already been tried and what attempts you have made to resolve the issue

Example (MCVE)

- [ ] Example provided has too much irrelevant code and is not considered minimal (*Minimal*)
- [ ] Example provided has undefined variables that make it impossible to reproduce (*Complete*)
- [x] Example provided is missing code necessary to reproduce the error (*Complete*)
- [x] When trying to reproduce example provided I did not receive the same error message as described in issue (*Verifiable*)
- [x] No example provided (*Example*)

Error

- [ ] No error included
- [ ] Error included is unclear or confusing
- [ ] No stack trace included in error
- [ ] No information about current and expected result

Read through our [contributing guidelines](https://github.com/dynamoose/dynamoose/blob/master/CONTRIBUTING.md) for more information about what is expected when submitting issues.

If you have any questions about why this message was posted on your issue, please comment below with your specific question and we'd be happy to discuss with you further. If you believe you have followed the guidelines, please comment your reasoning below for why you believe you have followed the guidelines checked above, and we'll take another look at it.

While we understand that these guidelines create a larger burden on you, it is critical you follow them. The project maintainers have to handle everyone's issues while you only have to write yours, so it is imperative you make our jobs easier and help save us time by following the guidelines.

In order to proceed, please create a *new issue* ensuring you address all guidelines checked above (while also taking note to not violate any guidelines you followed).

---

Additional Information:

Still can't reproduce with the following code:

```js
(async () => {
	const dynamoose = require("dynamoose");

	const schema = new dynamoose.Schema({
		"id": String
	});

	const Model = dynamoose.model(`Test-${Date.now()}`, schema);

	new dynamoose.Table(`Test-${Date.now()}`, [Model], {
		"waitForActive": true,
		"create": true,
		"update": true
	});

	console.log("Done");
})();
```

---

Disclosure: This was based on a [GitHub Saved Reply](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/working-with-saved-replies). The checkboxes were manually checked by myself based on your original issue.

**Comment by fishcharlie** [comment]
@Drarig29 I was finally able to reproduce this, I think it should be fixed in #1453. No idea why I couldn't reproduce it with my example above...

**Comment by Drarig29** [comment]
Yes. I even had trouble reproducing the issue locally in a blank project, so I had to comment out some code in my project instead...

Hopefully it will be fixed!
Thanks for trying even after you closed the issue
